pkg/vex/repo/repo.go

```diff
@@ -0,0 +1,328 @@
+package repo
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"net/url"
+	"os"
+	"path"
+	"path/filepath"
+	"time"
+
+	"github.com/hashicorp/go-getter"
+	"github.com/samber/lo"
+	"golang.org/x/xerrors"
+
+	"github.com/aquasecurity/trivy/pkg/clock"
+	"github.com/aquasecurity/trivy/pkg/downloader"
+	"github.com/aquasecurity/trivy/pkg/log"
+	"github.com/aquasecurity/trivy/pkg/utils/fsutils"
+)
+
+const (
+	SchemaVersion = "0.1"
+
+	manifestFile      = "vex-repository.json"
+	indexFile         = "index.json"
+	cacheMetadataFile = "cache.json"
+)
+
+type Manifest struct {
+	Name          string    `json:"name"`
+	Description   string    `json:"description"`
+	Versions      []Version `json:"versions"`
+	LatestVersion string    `json:"latest_version"`
```

Comment:
vex-repo-spec doesn't contain this field- https://github.com/aquasecurity/vex-repo-spec/blob/main/vex-repository.schema.json

Reply:
Thanks. Deleted https://github.com/aquasecurity/trivy/pull/7206/commits/424759ed76d159674b319282498f02c59fac3c81